spaces/domains.json

```
@@ -1,4 +1,6 @@
 {
+  "vote.enerart.com": "enerart.eth",
+  "vote.token.pro": "enerart.eth",
```

You will need to choose only one subdomain per ENS.
```suggestion
  "vote.enerart.com": "enerart.eth",
```

---

ok
please add "vote.enerart.com": "enerart.eth"